kasa/cli.py

```diff
@@ -617,12 +622,17 @@ async def raw_command(ctx, dev: SmartDevice, module, command, parameters):
 @click.option("--module", required=False, help="Module for IOT protocol.")
 @click.argument("command")
 @click.argument("parameters", default=None, required=False)
-async def cmd_command(dev: SmartDevice, module, command, parameters):
+async def cmd_command(dev: Device, module, command, parameters):
     """Run a raw command on the device."""
     if parameters is not None:
         parameters = ast.literal_eval(parameters)
 
-    res = await dev._query_helper(module, command, parameters)
+    if isinstance(dev, IotDevice):
+        res = await dev._query_helper(module, command, parameters)
```

Comment:
We should probably add a method for this use-case to the interface, but that's something for a separate PR.